src/pages/nerdpacks.js

```diff
@@ -67,7 +67,7 @@ const NerdpackPage = ({ data }) => {
           <iframe
             width="578"
             height="328.125"
-            src="https://www.youtube.com/embed/ZagZfNQYJEU?controls=0"
+            src="https://youtu.be/7omo0qHxku8"
```

Comment:
@veextee This link needs to be the `embed code` link for it to work properly:
```suggestion
            src="https://www.youtube.com/embed/7omo0qHxku8"
```

I'll make that change and get this pushed out 👍